environments/icds-cas/inventory/postgresql.csv

```diff
@@ -11,25 +11,25 @@ pgmainstandby0,100.71.184.28,pg_standby,main_dbs,MUMGCCWCDPRDPMS0,,pgmain0,,stan
 pgmainstandby1,100.71.184.82,pg_standby,main_dbs,MUMGCCWCDPRDPMS1,,pgmain0,,standby1
 pgmainstandby2,100.71.184.85,pg_standby,main_dbs,MUMGCCWCDPRDPMS2,,pgmain0,,standby2
 pgshard0,100.71.184.27,postgresql,shard_dbs,MUMGCCWCDPRDPS00,pgshard0standby,,"[""standby0"",""spare""]",
-pgshard1,100.71.184.29,postgresql,shard_dbs,MUMGCCWCDPRDPS01,pgshard1standby,,"[""standby0"",""spare""]",
+pgshard1,100.71.184.29,pg_standby,shard_dbs,MUMGCCWCDPRDPS01,,pgshard1standby,,standby0
```

Comment:
Are we not swapping the names?

Reply:
I don't think we should swap the names so that we keep the DD metrics consistent per VM.